### PR TITLE
{2023.06}[system] EasyBuild v4.9.2

### PR DESCRIPTION
**Pull request by bedroge**

### CODE REVIEW

**Comment by bedroge** [comment]
bot: build repo:eessi.io-2023.06-software arch:aarch64/generic

**Comment by bedroge** [comment]
bot: build repo:eessi.io-2023.06-software arch:aarch64/generic

**Comment by bedroge** [comment]
bot: build repo:eessi.io-2023.06-software arch:aarch64/generic
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_n1
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_v1
bot: build repo:eessi.io-2023.06-software arch:x86_64/generic
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen2
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen3
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/haswell
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/skylake_avx512

**Comment by bedroge** [comment]
bot: build repo:eessi.io-2023.06-software arch:aarch64/generic

**Comment by bedroge** [comment]
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_n1
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_v1
bot: build repo:eessi.io-2023.06-software arch:x86_64/generic
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen2
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen3
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/haswell
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/skylake_avx512

File: easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.2-001-system.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+easyconfigs:
+  - EasyBuild-4.9.2.eb:
+      options:
+        from-pr: 20801
```
Comment (boegel) [inline easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.2-001-system.yml:4]: Use https://github.com/easybuilders/easybuild-easyconfigs/pull/20818 instead, and use EasyBuild v4.9.1 to install EasyBuild v4.9.2

**Review by boegel — CHANGES_REQUESTED** [review]

File: easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.2-001-system.yml (hunk 1 above)
Comment (bedroge) [reply easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.2-001-system.yml]: done in https://github.com/EESSI/software-layer/pull/604/commits/158f0eb96d2027a51bb8d5ad8ae347405790c8d4

**Comment by bedroge** [comment]
bot: build repo:eessi.io-2023.06-software arch:aarch64/generic
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_n1
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_v1
bot: build repo:eessi.io-2023.06-software arch:x86_64/generic
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen2
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen3
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/haswell
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/skylake_avx512

**Comment by boegel** [comment]
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen4

**Comment by trz42** [comment]
> bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen4

This should not build anything because the modified easystack file is not in the `zen4` subdirectory?

**Comment by boegel** [comment]
> > bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen4
> 
> This should not build anything because the modified easystack file is not in the `zen4` subdirectory?

I don't think there's anything enforcing that, that's just a convention?

**Comment by trz42** [comment]
> > > bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen4
> > 
> > 
> > This should not build anything because the modified easystack file is not in the `zen4` subdirectory?
> 
> I don't think there's anything enforcing that, that's just a convention?

You're right. 😀

File: easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.1-001-system.yml [hunk 2]
```diff
@@ -3,3 +3,7 @@ easyconfigs:
       options:
         from-pr: 20299
   - EESSI-extend-2023.06-easybuild.eb
+easyconfigs:
```
Comment (boegel) [inline easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.1-001-system.yml:6]: @bedroge You want to remove this line?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.1-001-system.yml (hunk 2 above)
Comment (bedroge) [reply easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.1-001-system.yml]: Oops, fixed.

**Comment by boegel** [comment]
all (9) staging PRs merged

**Review by ocaisa — APPROVED** [review]

**Review by boegel — APPROVED** [review]

**Pull request merged**